client/clientservice/src/event_service.cpp

```diff
@@ -52,9 +52,11 @@ Status EventServiceImpl::StreamEventGroups(ServerContext* context,
   auto span = opentracing::Tracer::Global()->StartSpan("stream_event_groups", {});
   client_->subscribe(request, span, callback);
 
-  while (!context->IsCancelled()) {
-  }
+  // TODO: Consider all gRPC return error codes as described in concord_client.proto
+  while (!context->IsCancelled())
```

Comment:
If `IsCancelled` does not use some sort of blocking, this code may potentially become a huge busy loop.

Reply:
This won't be an empty loop. It will be filled with the TRC integration.

Reply:
I am afraid to leave such code as-is: if you say that it will be filled later then put a sleep in the while for now.